metricbeat/module/rabbitmq/metricset_test.go

```diff
@@ -47,9 +48,12 @@ func newTestMetricSet(base mb.BaseMetricSet) (mb.MetricSet, error) {
 }
 
 // Fetch makes an HTTP request to fetch connections metrics from the connections endpoint.
-func (m *testMetricSet) Fetch() ([]common.MapStr, error) {
+func (m *testMetricSet) Fetch(reporter mb.ReporterV2) {
```

Comment:
Just curious, why we have a fetch function here in the test?

Reply:
Yeap, this is weird. I have no idea. Maybe it's related with the fact that this modules uses mocked data https://github.com/elastic/beats/tree/master/metricbeat/module/rabbitmq/_meta/testdata